packages/opencensus-instrumentation-http/src/http.ts

```diff
@@ -226,66 +275,75 @@ export class HttpPlugin extends classes.BasePlugin {
         return request;
       }
 
-      // tslint:disable:no-any
-      const req = request.on('response', (response: any) => {
+      request.on('response', (response: http.ClientResponse) => {
         plugin.tracer.wrapEmitter(response);
         plugin.logger.debug('outgoingRequest on response()');
 
         response.on('end', () => {
           plugin.logger.debug('outgoingRequest on end()');
-          span.addAttribute('http.host', args[0].host);
-          span.addAttribute('http.method', request.method);
-          span.addAttribute('http.path', args[0].pathname);
-          span.addAttribute('http.route', args[0].path);
+          const method = response.method ? response.method : 'GET';
+          const reqUrl = url.parse(args[0]);
+          span.addAttribute(plugin.ATRIBUTE_HTTP_HOST, reqUrl.hostname);
+          span.addAttribute(plugin.ATRIBUTE_HTTP_METHOD, method);
+          span.addAttribute(plugin.ATRIBUTE_HTTP_PATH, reqUrl.pathname);
+          span.addAttribute(plugin.ATRIBUTE_HTTP_ROUTE, reqUrl.path);
+          span.addAttribute(plugin.ATRIBUTE_HTTP_USER_AGENT, null);
```

Comment:
Looks like the user agent disappeared here? Is this intentional?

Reply:
A refactore was made.